compiler/il/DataTypes.hpp

```diff
@@ -24,7 +24,7 @@
 namespace TR
 { 
 
-class DataType : public OMR::DataTypeConnector
+class OMR_EXTENSIBLE DataType : public OMR::DataTypeConnector
```

Comment:
hm, I don't see a3ffa2d1fedd8636736dcad19853e3f5d2888624 having been updated to address the comment. I noticed you changed the one for Symbol (thanks!), but this one is for DataType.